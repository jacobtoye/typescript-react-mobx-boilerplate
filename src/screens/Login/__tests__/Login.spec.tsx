import * as React from 'react';
import { shallow } from 'enzyme';

import { Login } from '../index';

describe('Login', () => {
  it('should render correctly', () => {
    const performLogin = jest.fn();
    const component = shallow(
      <Login performLogin={performLogin} />,
    );
    expect(component).toMatchSnapshot();
  });
});
