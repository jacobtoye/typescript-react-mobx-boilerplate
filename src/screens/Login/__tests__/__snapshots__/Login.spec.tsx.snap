// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render correctly 1`] = `
<div
  className="login"
>
  <Component
    onClick={[MockFunction]}
  >
    Login
  </Component>
</div>
`;
